pkg/exchange/binance/util.go

```diff
@@ -0,0 +1,17 @@
+package binance
+
+import (
+	"crypto/ed25519"
+	"encoding/base64"
+	"fmt"
+	"time"
+)
+
+func generateSignature(apiKey string, privateKey ed25519.PrivateKey) (string, int64) {
```

Comment:
extract time.Now() as a function parameter, so that you can add a test against this function